src/mapgen/mapgen.h

```diff
@@ -30,10 +30,8 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #define MAPGEN_DEFAULT_NAME "v7"
 
 /////////////////// Mapgen flags
-#define MG_TREES       0x01  // Obsolete. Moved into mgv6 flags
```

Comment:
comment instead of remove, it prevent bad use in the future

Reply:
These flags are internal, they can removed and their values changes at will.
So these lines should be removed.

Reply:
you mean all of them or just the ones I commented?

Reply:
the lines you commented should be removed